packages/xchain-thorchain/src/client.ts

```diff
@@ -233,6 +233,15 @@ class Client implements ThorchainClient, XChainClient {
       average: baseAmount(0, 6),
     })
   }
+
+  getDefaultFees = (): Fees => {
+    return {
+      type: 'base',
+      fast: baseAmount(750, 6),
+      fastest: baseAmount(2500, 6),
+      average: baseAmount(0, 6),
+    }
+  }
```

Comment:
BTW: Just saw we use in [getBalance](https://github.com/xchainjs/xchainjs-lib/blob/a0d41fe2ad832feacc632cbe3fd89ddc23ab927a/packages/xchain-thorchain/src/client.ts#L152) decimal of `8` (not `6` like in `getDefaultFees'). What's the correct decimal for `Thorchain`?